packages/fast-components-styles-msft/src/toggle/index.ts

```diff
@@ -0,0 +1,151 @@
+import {IDesignSystem, hexToRGB} from "../design-system";
+import {ComponentStyles} from "@microsoft/fast-jss-manager";
+import {IToggleClassNameContract} from "@microsoft/fast-components-class-name-contracts";
+
+const styles: ComponentStyles<IToggleClassNameContract, IDesignSystem> = {
+    toggle: {
+        '& > label': {

```

Comment:
I think we we include label here we will want to add a managed class for it rather than using this format. We'll likely want to follow that principle for all other elements as well

Reply:
Not sure I follow. Why would this need a managed class and none of the other elements

Reply:
I would, for the most part that any of the elements in here needing styles to have a managed class name. The name can be part of the contract for toggle. There will be additional gains here as well by using basic classes and not having the complex selector chains.